package.json

```diff
@@ -93,6 +93,8 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js",
```

Comment:
modify this so that it runs through your test:it step (like the other apps)

Reply:
@arcepaul modified!